get_pull_request_title.rb

```diff
@@ -3,13 +3,16 @@
 
 if ARGV.length != 1 then
   puts "Expected only one argument: ENDPOINT_URL"
-  exit -1
+  exit 1
 end
 
-responce = Net::HTTP.get_response(URI(ARGV[0]))
-if responce.code == "200"
-  puts JSON.parse(responce.body)["title"]
-else
-  puts "Had an issue calling endpoint..."
-  exit -1
+for i in 0..10
+  response = Net::HTTP.get_response(URI(ARGV[0]))
+  if response.code == "200"
+    puts JSON.parse(response.body)["title"]
+    exit 0
+  end
 end
+
+puts "Had an issue calling endpoint...%s %s %s" % [response, response.message, response.body]
+exit 1
```

Comment:
Add a new line at the end of the file, please.
```suggestion
exit 1

```